### ISSUE TITLE
Unable to get image in mail after sending in request

### ISSUE THREAD

**Issue by ashuags2** [caronc/apprise-api#49]
:mega: **Notification Service(s) Impacted**

:beetle: **Describe the bug**
Here is my code - 

Request -
{
 "urls":"mailtos://brokername:abc@gmail.com/?smtp=smtp.gmail.com&name=Broker&to=abc@gmail.com",
 "title":"test",
 "body":"hello",
 "attach":"https://i.redd.it/my2t4d2fx0u31.jpg?name=FlyingToMars.jpg"
}

:bulb: **Screenshots and Logs**

:computer: **Your System Details:**
 - OS: [e.g. RedHat v8.0]
 - Python Version: [e.g. Python v2.7]

:crystal_ball: **Additional context**

**Comment by caronc** [comment]
Hi @ashuags2 unfortunately there is no way to pass along attachments at this time through the sidecar (or at least i haven't thought of a good way to do this).

The problem is that the URL you specify (in your example `https://i.redd.it/my2t4d2fx0u31.jpg?name=FlyingToMars.jpg`) causes apprise (the underlining tool) actually does a remote request (storing it locally) to-be attached.  Attachment support gets a little hairy too if you were to specify `file://` (instead of `http://` as then Apprise would reference the local file-system). If the functionality was added, it would have to be clever enough to ignore `file://` requests (so no one requests `file:///etc/passwd` or something like that).  

Maybe light attachment support via URLs only could be added to the container in the future.  I suppose it could write the temporary (retrieved files to-be-attached) files it uses back to the same storage exported for the configuration files.

I'd have to ponder this one

**Comment by ashuags2** [comment]
Hello @caronc i tried like this -
"attach":"file://C:/Users/windev/Downloads/userdocs/photo.jpg"
and this photo file is present on this location. still i am not getting that photo image on my mail.
can you please tell me the way that how can i pass local storage image?

**Comment by caronc** [comment]
Yeah, you're not doing anything wrong.  The point is that the API does not relay attachments. It does not accumulate files in any way.  If you want to send an attachment, you need to use the `apprise` CLI tool (found [here](https://github.com/caronc/apprise)
